scapy/contrib/ikev2.py

```diff
@@ -99,6 +101,19 @@
                                                  "ESN":   1,  }, 0),
                          }
 
+IKEv2AuthenticationTypes = {
+  0 : "Reserved",
+  1 : "RSA Digital Signature",
```

Comment:
Sure.
When I fix an old patch, i useally leave the old work in a single piece. I'll add the commit just after that.